pandas/tests/io/test_parquet.py

```diff
@@ -931,6 +931,14 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    def test_read_zero_chunked_array(self, pa):
+        df = pd.DataFrame({"value": pd.array([], dtype=pd.Int64Dtype())})
+        print(pa)
+        with tm.ensure_clean() as path:
```

Comment:
check_round_trip does this

Reply:
I change check_roud_trip to tm.assert_frame_equal

Reply:
no my point is you *can* use check_round_tirp and you don't need L936-939